modules/ROOT/pages/to-describe-an-asset.adoc

```diff
@@ -4,50 +4,92 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-. Click an asset.
-. Click Edit to add a description. The editor lets you change the appearance of text,
-insert code samples, block quotes, and horizontal lines. You can also insert a link
-to an external resource and to an image. The image has to be served on the web.
-+
-The Markdown editor also lets you add a link to a YouTube or Vimeo video to your asset description.
-+
-. After you create a page, you can add additional pages, tags, terms and conditions,
-and change versions for an asset or if describing an API, for versioning the API as well.
+You can edit an asset's portal, change its name, icon, description, and tags, and add portal pages with content.
 
-== Change an Asset Title
+In Exchange, open the asset list and select the asset. Exchange shows the asset portal.
 
-. Position your mouse pointer in the asset's title and add the text you want the title to be.
-. Press the Enter (or Return) key on your keyboard.
-+
-If you don't press the Enter (Return) key, the title reverts to its former text.
-+
-If you edit the title and click out of the edit field rather than press Enter (Return), the title does not display your changes. Click back in the title field to see your changes. If you publish the page without pressing Enter, you don't see your changes. Your changes are only present in the field when you click in the title field again, but they are not made visible until you press Enter (Return).
+== Edit Asset Name
+
+. Select the asset name or the pen icon after the asset name.
+. Enter the new asset name.
+. (Optional) To cancel the changes, select the cancel icon.
+. To save the new name, press the enter or return key on your keyboard, or select the check icon.
+
+NOTE: Before the August 2019 release, the versions of an asset could have different names. After the August 2019 release, a new asset name applies to all versions of the asset.
+
+
+== Edit Asset Icon
+
+. Select the pen icon beside the asset icon.
+. Select an image.
+. (Optional) To remove an asset icon, select the cancel icon beside the asset icon.
+
+NOTE: Before the August 2019 release, the versions of an asset could have different icons. After the August 2019 release, a new asset icon applies to all versions of the asset.
+
+
+== Edit Asset Description
+
+. Select the pen icon after the asset description.
+. Enter the new asset description.
+. (Optional) To cancel the changes, select the cancel icon.
+. To save the new name, select the check icon.
+
+NOTE: Before the August 2019 release, the versions of an asset could have different descriptions. After the August 2019 release, a new asset description applies to all versions of the asset.
```

Comment:
Before the August 2019 release, we didn't even have Asset Description field, right? We should update the text to reflect that - @LucasLudueno , @mcampo

Reply:
Changed to just "NOTE: A new asset description applies to all versions of the asset."

Reply:
right. Is it necessary to add the word `new`? Why not just `NOTE: The asset description applies to all versions of the asset`? (just another doubt)

Reply:
Good point! Fixed this.